lib/spack/spack/installer.py

```diff
@@ -1945,6 +1971,7 @@ def run(self):
             spack.hooks.post_install(self.pkg.spec)
 
         _print_timer(pre=self.pre, pkg_id=self.pkg_id, timer=self.timer)
+        _print_install_test_log(self.pkg, self.verbose)
```

Comment:
I feel like this should come before the timer, and possibly be included in the time calculation. Maybe an extra step for how long the tests took to run.

Reply:
I ended up adding a timer for handling `post-install`, which may be more than just the tests.

Just noticed I don't log `Completed` for post-install tests like I do when running stand-alone tests just for the purposes of extracting test timing from the test log (for CDash reporting).

Reply:
Added `Completed testing` when running a `test` method post-install.